week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -28,10 +28,31 @@ values will have a remainder of 0 i.e. there are some odd numbers in the array t
 
 /* Now try to complete the exercise */
 
-function bushChecker() {
+function bushChecker(berries) {
+
+  let convertTonumber = berries.map(function (berry){
+    return berry.charCodeAt();
+  });
+
+  function findPinkBerries(convertedArr){
+    let safeToEat = convertedArr.every((berry) => {return berry === 112});
```

Comment:
```suggestion
    let safeToEat = convertedArr.every((berry) => berry === 112);
```
You can use an implicit return here with an arrow function if an expression like `berry === 112` is on a single line.